parquet/compress/compress_test.go

```diff
@@ -138,3 +138,23 @@ func TestCompressReaderWriter(t *testing.T) {
 		})
 	}
 }
+
+func TestMarshalText(t *testing.T) {
+	compression := compress.Codecs.Zstd
+	data, err := compression.MarshalText()
+	assert.NoError(t, err)
+	assert.Equal(t, "ZSTD", string(data))
+}
+
+func TestUnmarshalText(t *testing.T) {
+	var compression compress.Compression
+	err := compression.UnmarshalText([]byte("ZSTD"))
+	assert.NoError(t, err)
+	assert.Equal(t, compress.Codecs.Zstd, compression)
+}
```

Comment:
I figured we should probably use a table test and test all the codes. i.e.

```go
func TestMarshalText(t *testing.T) {
    tests := []struct{
            codec ....
            text string
    }{
         {compress.Codecs.Zstd, "ZSTD"},
         {compress.Codecs.Snappy, "SNAPPY"},
          ...
      }

      for _, tt := range tests {
          t.Run(tt.text, func(t *testing.T) {
               data, err := tt.codec.MarshalText()
               assert.NoError(t, err)
               assert.Equal(t, tt.text, string(data))
               ...
          })
      }
}
```

and so on